pkg/lockfile/lockfile.go

```diff
@@ -13,13 +13,20 @@ import (
 // by the lock.
 type Locker interface {
 	// Acquire a writer lock.
+	// The default unix implementation panics if:
+	// - opening the lockfile failed
+	// - tried to lock a read-only lock-file
 	Lock()
```

Comment:
Theoretically we would have to add those notes to every method of `Store` as well but I think this is somehow overkill. :thinking: